test-configs.yaml

```diff
@@ -179,7 +180,57 @@ test_plans:
         kms_atomic
         kms_flip_event_leak
         kms_prop_blob
-        kms_setmode kms_vblank
+        kms_setmode
+        kms_vblank
+
+  kms-tegra:
+    rootfs: debian_buster-igt_ramdisk
+    pattern: 'igt/{category}-{method}-{protocol}-{rootfs}-igt.jinja2'
+    params:
+      drm_driver: "tegra"
+      tests: >
+        core_auth
+        core_getclient
+        core_getstats
+        core_getversion
+        core_setmaster_vs_auth
+        drm_read
+        kms_addfb_basic
+        kms_atomic
+        kms_flip_event_leak
+        kms_prop_blob
+        kms_setmode
+        kms_vblank
+
+  kms-exynos:
+    rootfs: debian_buster-igt_ramdisk
+    pattern: 'igt/{category}-{method}-{protocol}-{rootfs}-igt.jinja2'
+    params:
+      drm_driver: "exynos"
+      tests: >
+        core_auth
+        core_getclient
+        core_getstats
+        core_getversion
+        core_setmaster_vs_auth
+        drm_read
+        kms_addfb_basic
+        kms_atomic
+        kms_flip_event_leak
+        kms_prop_blob
+        kms_setmode
+        kms_vblank
+
+  gpu-panfrost:
```

Comment:
The `panfrost` driver was merged in v5.2, so I think there should be a kernel version blacklist to not test earlier stable ones (i.e. up to v4.19).  This is especially important if we add checks for the presence of the device as part of the test plan, since it would result in a test case failure (we've added a device presence test case to v4l2-compliance to detect when the driver failed to probe).

Reply:
I think this should be fine, igt will just skip the test if a suitable device isn't found.

Reply:
OK well we can always add filters later if this becomes an issue.